src/org/labkey/test/tests/ParsingPatternForDateTest.java

```diff
@@ -44,16 +65,52 @@ private void doSetup() throws IOException, CommandException
         goToProjectHome();
         PortalHelper portalHelper = new PortalHelper(this);
         portalHelper.addWebPart("Lists");
+    }
+
+    @Before
+    public void resetForTest()
+    {
+        resetSiteSettings();
+        resetProjectSettings();
+
+        log("Verify no patters are set for the project.");
+        verifyNoPatternsSet(ProjectSettingsPage.beginAt(this, getProjectName()));
 
-        FieldDefinition.LookupInfo lookupInfo = new FieldDefinition.LookupInfo(getProjectName(), "lists", dateList);
-        TestDataGenerator dgen = new TestDataGenerator(lookupInfo)
-                .withColumns(List.of(
-                        new FieldDefinition("name", FieldDefinition.ColumnType.String),
-                        new FieldDefinition("dateCol", FieldDefinition.ColumnType.DateAndTime)));
-        dgen.createDomain(createDefaultConnection(), "IntList", Map.of("keyName", "id"));
+        log("Verify no patters are set for the site.");
+        verifyNoPatternsSet(LookAndFeelSettingsPage.beginAt(this));
+
+    }
+
+    private void resetSiteSettings()
+    {
+        log("Reset site settings.");
+        try
+        {
+            resetSettings("/");
+        }
+        catch (IOException | CommandException e)
+        {
+            TestLogger.error("Failed to reset site look and settings.", e);
+        }
+    }
+
+    private void resetProjectSettings()
+    {
+        log("Reset project settings.");
+        try
+        {
+            resetSettings(getProjectName());
+        }
+        catch (IOException | CommandException e)
+        {
+            TestLogger.error("Failed to reset project settings.", e);
+        }
```

Comment:
Rethrow this instead of ignoring it.
```suggestion
        catch (IOException | CommandException e)
        {
            throw new RuntimeException("Failed to reset project settings.", e);
        }
```

Reply:
Changed.